schematic/utils/general.py

```diff
@@ -69,4 +69,31 @@ def convert_gb_to_bytes(gb: int):
     return: total number of bytes
     """
     bytes_to_return = gb * 1024 * 1024 * 1024
-    return bytes_to_return
\ No newline at end of file
+    return bytes_to_return
+
+
+def entity_type_checking(syn, manifest_id):
```

Comment:
Nit: What are you doing in this function? 

Are you checking the entity type?  Or are you mapping the entity type to a string?

Reply:
I am trying to return the entity type, and I forgot to change the name (after copying it from an old function). I will update the name of it. Thanks for pointing it out.